concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-234	234	éste	this	1214	THIS
+Lastra-1986-381-235	235	ése	that	78	THAT
+Lastra-1986-381-236	236	aquél	that	78	THAT
+Lastra-1986-381-237	237	así	So		
+Lastra-1986-381-238	238	aquí	here	136	HERE
+Lastra-1986-381-239	239	cerca	near	1942	NEAR
+Lastra-1986-381-240	240	hoyo	hole		
+Lastra-1986-381-241	241	pasado mañana	day after tomorrow	1179	DAY AFTER TOMORROW
+Lastra-1986-381-242	242	antier	the day before yesterday	1180	DAY BEFORE YESTERDAY
+Lastra-1986-381-243	243	temprano ('en la mañana temprano')	Early ('early morning')		
+Lastra-1986-381-244	244	temprano ('que todavía hay mucho tiempo')	Early ('There's a lot of time')	672	EARLY
+Lastra-1986-381-245	245	tarde ('en la tarde')	afternoon ('in the afternoon')	1807	AFTERNOON
+Lastra-1986-381-246	246	tarde ('que ya no hay tiempo suficiente')	afternoon ('that there is no enough time')	477	LATE
+Lastra-1986-381-247	247	mucho (trabaja mucho)	Much (work hard)	3452	MUCH
+Lastra-1986-381-248	248	muchos (hombres)	many men)	1198	MANY
+Lastra-1986-381-249	249	todo (quiero todo)	Everything (I want everything)	98	ALL
+Lastra-1986-381-250	250	todos (todos los hombres)	All (all men)	98	ALL
+Lastra-1986-381-251	251	¿qué?	that?	1236	WHAT
+Lastra-1986-381-252	252	¿quién?	who?	1235	WHO
+Lastra-1986-381-253	253	¿por qué?	why?	1270	WHY
+Lastra-1986-381-254	254	¿cuántos?	how many?	822	HOW MANY
+Lastra-1986-381-255	255	(preguntar cuál es el saludo)	(Ask what is the greeting)		
+Lastra-1986-381-256	256	¡buenos días!	good morning!		
+Lastra-1986-381-257	257	¡buenas tardes!	Good afternoon!		
+Lastra-1986-381-258	258	yo	I	1209	I
+Lastra-1986-381-259	259	tú	you (informal)	1215	THOU
+Lastra-1986-381-260	260	usted	you (formal)	1215	THOU
+Lastra-1986-381-261	261	nosotros	we	1212	WE
+Lastra-1986-381-262	262	ustedes	you	1213	YOU
+Lastra-1986-381-263	263	él	he	1211	HE
+Lastra-1986-381-264	264	ellos	they	817	THEY
+Lastra-1986-381-265	265	¡bebe! (preguntar si hay diferencia entre beber agua y beber licor)	baby! (Ask if there is a difference between drinking water and drinking liquor)		
+Lastra-1986-381-266	266	estoy comiendo	I'm eating		
+Lastra-1986-381-267	267	estamos comiendo	we are eating		
+Lastra-1986-381-268	268	están comiendo (ellos)	They are eating (them)		
+Lastra-1986-381-269	269	comió	he ate		
+Lastra-1986-381-270	270	comieron	They ate		
+Lastra-1986-381-271	271	tengo sed	I'm thirsty		
+Lastra-1986-381-272	272	tengo hambre	I'm hungry		
+Lastra-1986-381-273	273	me mordió (un perro)	it bite me (a dog)		
+Lastra-1986-381-274	274	nos mordió (un perro)	it bite us (a dog)		
+Lastra-1986-381-275	275	olió (= olfatear)	it smelled		
+Lastra-1986-381-276	276	olieron	They smelled		
+Lastra-1986-381-277	277	vomitó	it threw up		
+Lastra-1986-381-278	278	se mareó ( = borracho)	it became dizzy (=drunk)		
+Lastra-1986-381-279	279	tengo frío	I'm cold		
+Lastra-1986-381-280	280	hace frío	it's cold		
+Lastra-1986-381-281	281	gruñe (la barriga)	it growls (the belly)		
+Lastra-1986-381-282	282	¡habla!	speak!	1623	SPEAK
```

Comment:
Is the exclamation here part of the concept as in something like `SPEAK (COMMAND)`? (likewise for 'roast it!' etc. below)

Reply:
Exactly. There was an issue opened by me about the mapping of imperatives (#1337). We decided that the best solution for imperative verbs (with their infinitive form absent on the list) should be to map them as their infinitives in Concepticon.

Reply:
Ah, right, thanks for reminding me. I had forgotten that discussion.